models.py

```diff
@@ -309,8 +318,10 @@ def update_wallet_address(cls, session, user):
                     if have_tx:
                         break
                     if tx["type"] == 4 and tx["assetId"] == app.config["ASSET_ID"]:
+                        #amount_nzd = round(((tx['amount']/100) * float( 1 - rate)),2)
+                        amount_nzd = (tx['amount'] / float(1 + rate))
```

Comment:
see https://github.com/zap-me/zapm_server/pull/21#discussion_r442557209 (move duplicated code to utility function in utils.py)

Reply:
so a new classmethod? that calls utils.py

Reply:
removed this line in app.py:
```
amount_receive = amount / (1 + merchant_rate)
```
replaced with:
```
amount_receive = amount_conversion_rate(amount, merchant_rate)
```

pushed the change